### PR TITLE
Add Launch GUI (Zenith)

### PR DESCRIPTION
**Pull request by akevinge**
### Main Changes
- Add Zenith GUI + server
- New Dockerfiles (see `zenith/README.md`)

### Misc Changes
- Renamed docker container names to their respective service names (for some reason `navigator` and `navigator_obc` had the same, clashing container names).

### CODE REVIEW

**Comment by jruths** [comment]
Sorry I missed the demo last week!  I'm looking forward to seeing this!  

Regarding the docker names... I guess it could be improper convention, but navigator was the name of the image and we had two services navigator_obc and navigator_carla.  This allows you to `docker compose build navigator` and build the image (once) for both `docker compose run navigator_obc` and `docker compose run navigator_carla`.  It seemed unnecessary to have two identical images named navigator_obc and navigator_carla.  Does that make sense?

**Comment by akevinge** [comment]
> Sorry I missed the demo last week! I'm looking forward to seeing this!
> 
> Regarding the docker names... I guess it could be improper convention, but navigator was the name of the image and we had two services navigator_obc and navigator_carla. This allows you to `docker compose build navigator` and build the image (once) for both `docker compose run navigator_obc` and `docker compose run navigator_carla`. It seemed unnecessary to have two identical images named navigator_obc and navigator_carla. Does that make sense?

In theory, it shouldn't make a difference because it `navigator_obc` and `navigator_carla` all use the same `navigator` image. For example:
image

I could just be misunderstanding something though.

File: docker-compose.yml [hunk 1]
```diff
@@ -1,116 +1,138 @@
...
+    build:
+      context: .
+      dockerfile: Dockerfile
+
+  navigator_carla:
+    privileged: true
+    stdin_open: true
+    tty:
+      true
+      #runtime: nvidia
```
Comment (danielv012) [inline docker-compose.yml:16]: uncomment and outdent

**Review by danielv012 — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (akevinge) [reply docker-compose.yml]: Done

**Review by danielv012 — APPROVED** [review]

**Pull request merged**